### DISCUSSION TITLE
v-deep selector (Vue.js) equivalent in TailwindCSS?

### DISCUSSION THREAD

**Discussion by Arturexe** [tailwindlabs/tailwindcss#10107] [Help]
I'm using the `v-deep` selector to style `tiptap`, a rich text editor. For that the `.ProseMirror` class has to be accessed like so (`SCSS`):

    editor-content {
        // ... styles
        &::v-deep(.ProseMirror) {
          // ... styles
        }
    }

But how do I style the `.ProseMirror` class with `TailwindCSS`? I can't add classes to it, at least I don't know how. Is it possible?

**Comment by 192121978** [comment]
I had the same problem.Have you found a solution?

**Comment by adamwathan** [comment]
Hey! You can use [arbitrary variants](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-variants) to do this:

```html
<div class="[&_.ProseMirror]:rounded-xl ...">
  <div id="editor"></div>
</div>
```
